### PR TITLE
[Finishes #95928316] setScreen block has a dropdown of ids

### PR DESCRIPTION
**Pull request by Bjvanminnen**

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
screendropdown

File: apps/src/applab/dropletConfig.js [hunk 1]
```diff
@@ -8,6 +10,19 @@ var COLOR_RED = '#F78183';
...
+/**
+ * Generate a list of screen ids for our setScreen dropdown
+ */
+function getScreenIds() {
+  var ret = $(".screen").map(function () {
+    return '"' + this.id + '"';
+  });
+
+  // Convert from jQuery's array-like object to a true array
+  return $.makeArray(ret);
```
Comment (joshlory) [inline apps/src/applab/dropletConfig.js]: Hmm `ret` is already an array for me when I tried this out in Chrome.  Is it needed for another browser?

File: apps/src/applab/dropletConfig.js [hunk 2]
```diff
@@ -8,6 +10,19 @@ var COLOR_RED = '#F78183';
 var COLOR_CYAN = '#4DD0E1';
 var COLOR_YELLOW = '#FFF176';
 
+/**
+ * Generate a list of screen ids for our setScreen dropdown
+ */
+function getScreenIds() {
```
Comment (joshlory) [inline apps/src/applab/dropletConfig.js]: Maybe this should go in `designMode.js`?  But no strong preference.

**Comment by joshlory** [comment]
LGTM! :+1:

File: apps/src/applab/dropletConfig.js (hunk 1 above)
Comment (Bjvanminnen) [inline apps/src/applab/dropletConfig.js]: It's an array-like object, but not actually an array. The difference is subtle, but droplet didn't seem to be able to handle the array like object.

From Jquery documentation:
Array-Like Object
Either a true JavaScript Array or a JavaScript Object that contains a nonnegative integer length property and index properties from 0 up to length - 1. This latter case includes array-like objects commonly encountered in web-based code such as the arguments object and the NodeList object returned by many DOM methods.

When a jQuery API accepts either plain Objects or Array-Like objects, a plain Object with a numeric length property will trigger the Array-Like behavior.

**Pull request merged**